src/core/ext/transport/chttp2/transport/hpack_parser.h

```diff
@@ -121,7 +124,7 @@ class HPackParser {
   uint8_t dynamic_table_updates_allowed_;
   // Length of frame so far.
   uint32_t frame_length_;
-  uint32_t metadata_size_limit_;
+  std::unique_ptr<RandomEarlyDetection> metadata_early_detection_ = nullptr;
```

Comment:
I don't think this needs a `unique_ptr`... let's just inline the type here.

Reply:
(like... `RandomEarlyDetection metadata_early_detection_;`)

Reply:
Done - I hadn't done this because there was no default constructor for RandomEarlyDetection, but I've added one now.